src/api/spec/db/data/backfill_notified_projects_spec.rb

```diff
@@ -39,13 +39,16 @@
     end
 
     it 'backfills the notifications_projects table with all projects from existing notifications' do
-      expect(NotifiedProject.pluck(:project_id)).to eq([
+      expected = NotifiedProject.pluck(:project_id).sort
```

Comment:
Why is this change necessary?

Reply:
Because the index somehow changed the order in which the results are returned and the spec didn't enforce any other order.

Reply:
Things like this need to be mentioned in the commit message...

Reply:
On it

Reply:
Done